tests/integration/standard/test_metadata.py

```diff
@@ -1332,7 +1332,7 @@ def test_token(self):
         self.assertEqual(expected_node_count, len(tmap.ring))
         cluster.shutdown()
 
-
+@scylla_only
```

Comment:
Can we have explanation why it is ran only on scylla ?

Reply:
added a comment, basically seems this feature didn't cover cassandra specific queries (i.e. peers_v2)